apps/server/lib/bootstrap.ts

```diff
@@ -302,6 +397,73 @@ export const bootstrap = async (logContext: LogContext, options: any) => {
 		}),
 	);
 
+	// Manually sync oauth clients
+	// TODO: This should ideally be completely automated, but this would
```

Comment:
I think we should probably have a sensible load priority for default contracts. Off the top of my head it should be:
1. types
2. relationships
3. actions
4. triggered actions
5. everything else

Reply:
@LucianBuzzo there is a problem with preinserting links, because you don't know ids of contracts which you are linking before they are inserted.